charts/agent-stack-k8s/values.yaml

```diff
@@ -18,3 +18,5 @@ annotations: {}
 secretsMetadata: {}
 
 serviceAccountMetadata: {}
+
+installGrafanaDashboard: true
```

Comment:
deploy or install? The values.schema.json uses `deploy` 😬

Reply:
Good catch, thanks!